sys/posix/pthread/pthread_spin.c

```diff
@@ -37,7 +39,7 @@ int pthread_spin_destroy(pthread_spinlock_t *lock)
         return EINVAL;
     }
 
-    (void) lock;
+    atomic_flag_clear(&(lock->flag));
```

Comment:
After reviewing the manual pages again (http://pubs.opengroup.org/onlinepubs/9699919799/functions/pthread_spin_destroy.html), I see that this change was not needed and should just be removed.